pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts

```diff
@@ -32,14 +44,14 @@ const toAPI = (xtradbCluster: XtraDBCluster): XtraDBClusterPayload => ({
     cluster_size: xtradbCluster.clusterSize,
     pxc: {
       compute_resources: {
-        cpu_m: xtradbCluster.cpu,
-        memory_bytes: xtradbCluster.memory * 1024,
+        cpu_m: xtradbCluster.cpu * 1000,
```

Comment:
Shouldn't those figure be multiples of 1024?

Reply:
I thought so too but we are using [DecimalSI](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L119) format on the backend. I tested with 2000000000 and I got 2Gb as expected like shown in the following image 
<img width="500" alt="Screenshot 2020-10-27 at 11 00 20" src="https://user-images.githubusercontent.com/16366502/97294243-a7aee880-1845-11eb-9ebc-05f686916bf4.png">